scripts/generate_markdown_readme.py

```diff
@@ -20,6 +23,35 @@
         raise SystemExit(1)
     raise (e)
 
+def process_commands(wdl_file):
+    dir_name = os.path.dirname(args.input_wdl_path)
+    command_file = "./commands.txt" if dir_name == "" else dir_name + "/commands.txt"
+
+    if not os.path.isfile(command_file):
+        with open(args.input_wdl_path, 'r') as w:
+            wdl_lines = w.readlines()
```

Comment:
replace lines 32 and 33 with:
multi_line = w.read()
for simplification

Reply:
yes, it looks cleaner now